requirements.txt

```diff
@@ -10,7 +10,7 @@ Flask-WTF
 Flask-Mail
 coloredlogs<=14.0
 query_phenomizer
-Flask-Babel
+Flask-Babel<3
```

Comment:
👍🏻

Reply:
Thanks for the review! At some point we may want to look into why and update, but we are in no rush and can defer this a bit and see what happens with that new version.